HW7/models/carts.model.ts

```diff
@@ -0,0 +1,80 @@
+import Cart, { ICart } from "./schemas/cart.schema.ts";
+
+import { CartEntity, CartItemEntity, ProductEntity } from "../types/index.ts";
+
+const getCart = (id: string): Promise<ICart | null> => {
+  return new Promise((resolve, reject) => {
```

Comment:
This looks very strange: you return promise which resolves synchronously with promise (Cart.findOne returns promise). Even though this works, I'd recommend to rewrite this:
```
const getCart = async (id: string): Promise<ICart | null> => {
  const cart = await Cart.findOne({ userId: id });

  return cart;
};
```

Reply:
Updated.
Thanks for review)